website/src/sprinkles.css.ts

```diff
@@ -41,7 +41,7 @@ export const themeClass = createTheme(theme, {
   },
   colors: {
     header: "#f7eeed",
-    button: "#f7eeed",
+    button: "black",
```

Comment:
Why are you changing a color in the base theme?

Reply:
Beforehand the button was the same color as the header, so making buttons and links have the button color meant they were invisible in the base theme. So to fix this, I tried changing this to black. I see though this doesn't work well with other elements that already used the button color.